### PR TITLE
Switch to Node.js 16 from Nodesource as that will be a dependency of Zammad 5.2 soon.

### PR DESCRIPTION
**Pull request by mgruner**
This change does not hurt for the stable branch as well, and affects only the builder image, not the runner.

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
Could you please also adjust this lines indent and version: https://github.com/zammad/zammad-docker-compose/blob/a341566510dc8c44993cb62ea490331273f47cdc/containers/zammad/Dockerfile#L40

**Comment by mgruner** [comment]
@monotek of course, done. Sorry about that, it was caused by an overly eager auto format feature of my editor.

File: containers/zammad/Dockerfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ LABEL org.label-schema.build-date="$BUILD_DATE" \
       org.label-schema.vcs-url="https://github.com/zammad/zammad" \
       org.label-schema.vcs-type="Git" \
       org.label-schema.vendor="Zammad" \
-    org.label-schema.schema-version="5.1.0" \
+      org.label-schema.schema-version="5.1.0" \
```
Comment (monotek) [inline containers/zammad/Dockerfile:40]:
```suggestion
      org.label-schema.schema-version="5.2.0" \
```

**Review by monotek — CHANGES_REQUESTED** [review]

File: containers/zammad/Dockerfile (hunk 1 above)
Comment (monotek) [inline containers/zammad/Dockerfile:40]: Maybe we can put the whole change in: https://github.com/zammad/zammad-docker-compose/pull/274 and adjust to 5.2 there?

Comment (mgruner) [reply containers/zammad/Dockerfile:40]: I could do that if needed, but actually this PR here can already be merged for stable. Node.js 16 works fine there, too. Then we have less changes later for 5.2. Let me know.

**Review by monotek — APPROVED** [review]

**Pull request merged**